cmd/livepeer/livepeer.go

```diff
@@ -162,11 +162,14 @@ func main() {
 
 	// Config file
 	_ = flag.String("config", "", "Config file in the format 'key value', flags and env vars take precedence over the config file")
-	ff.Parse(flag.CommandLine, os.Args[1:],
+	err = ff.Parse(flag.CommandLine, os.Args[1:],
 		ff.WithConfigFileFlag("config"),
 		ff.WithEnvVarPrefix("LP"),
 		ff.WithConfigFileParser(ff.PlainParser),
 	)
+	if err != nil {
+		glog.Fatal("Error using config file: ", err)
```

Comment:
IIUC this error message will be not be accurate if CLI flags are used instead of a config file since `ff.Parse()` does not necessarily parse a config file if one is not provided and CLI flags are passed instead. Perhaps a more descriptive error message would be just: "Error parsing config"? WDYT?

Reply:
Right, updated in 3c0e0d3aab9ed3135c3b248ffba20380416067bc